src/dynamo-streams.ts

```diff
@@ -159,13 +159,18 @@ export class DynamoStreamHandler<Entity, Context> {
    */
   lambda(): DynamoDBStreamHandler {
     return async (event, ctx) => {
-      // 1. Handle the health check.
+      // 1. Handle potential health checks.
       if ((event as any).httpMethod) {
         return {
           statusCode: 200,
           body: JSON.stringify({ healthy: true }),
         } as unknown as void;
       }
+
+      if ((event as any).healthCheck) {
+        return { healthy: true } as any;
```

Comment:
I am not expert here and I just know about challenge service but it seems for health check we return the event, something like this:
```
if (isHealthCheck(event)) {
  return event;
}
```

what do you think?

Reply:
@baherhedayati good question -- it does not really matter what we return. The logic that invokes the health check just checks that the lambda did not throw an exception.